### ISSUE TITLE
syntax 'classProperties' isn't currently enabled

### ISSUE THREAD

**Issue by niksseif** [btholt/complete-intro-to-react-v5#22]
Hi,
I am getting this error <br>
**syntax 'classProperties' isn't currently enabled** <br>
although I have updated my babelrc file and I have added this to babelrc and removed dist and .cache <br>
`
{
  "presets": ["@babel/preset-react", "@babel/preset-env"],
  "plugins": ["@babel/plugin-proposal-class-properties"]
}`
<br>
and this is my dev dependencies in my package json!
<br>
`"devDependencies": {
    "@babel/core": "^7.0.0-0",
    "@babel/plugin-proposal-class-properties": "^7.4.4",
    "@babel/plugin-syntax-async-generators": "^7.2.0",
    "@babel/preset-env": "^7.4.3",
    "@babel/preset-react": "^7.0.0",
    "babel-eslint": "^10.0.2",
    "cross-env": "^5.2.0",
    "eslint": "^5.12.1",
    "eslint-config-prettier": "^4.0.0",
    "eslint-plugin-import": "^2.17.3",
    "eslint-plugin-jsx-a11y": "^6.2.1",
    "eslint-plugin-react": "^7.13.0",
    "eslint-plugin-react-hooks": "^1.0.2",
    "parcel-bundler": "^1.12.1",
    "prettier": "^1.16.1"
  },`
<br>
but I am still getting this error from the parcel

**Comment by ccherot** [comment]
I had a weird issue with this plugin as well.  When I ran the dev script that calls "parcel src/index.html" I would get an error saying that the plugin could not be found and the script proceeded to install an old version of babel and then it got stuck in a loop.  

After much mcuking around I was able to get parcel to see the plugin by removing the entry from .bablerc, then running the dev script and then when the dev script paused because of an error related to the use of class properties, I then added the plugin-proposal-class-properties entry back into the .babelrc.  It seems parcel was able to detect the change in the .babelrc file and automatically ran again and was able to see the plugin somehow.  

It was a huge pain to figure this out and I don't know if its an issue with Babel 7 or Parcel
